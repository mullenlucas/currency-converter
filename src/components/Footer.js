import { Link } from 'react-router-dom';
import '../css/footer.css';

function Footer() {
  return (
    <footer>
      <p>
        Powered by
        <Link to="https://www.amdoren.com">Amdoren</Link>
      </p>
    </footer>
  );
}

export default Footer;
