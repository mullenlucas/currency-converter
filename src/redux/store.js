// import { configureStore } from '@reduxjs/toolkit';

// const store = configureStore({
//   reducer: {
//     convert: convertReducer,
//   },
// });

// export default store;
