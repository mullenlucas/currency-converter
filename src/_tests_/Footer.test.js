import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../redux/store';
import Footer from '../components/Footer';

describe('Footer rendering test', () => {
  test('Footer snapshot rendering...', () => {
    const tree = render(
      <BrowserRouter>
        <Provider store={store}>
          <Footer />
        </Provider>
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
