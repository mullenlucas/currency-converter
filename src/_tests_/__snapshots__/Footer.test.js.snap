// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer rendering test Footer snapshot rendering... 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer>
        <div
          class="footer-header"
        >
          <p>
            Lucas Mullen
          </p>
          <p>
            All rights reserved
          </p>
        </div>
        <div
          class="footer-footer"
        >
          <p>
            Go to
            <a
              href="https://github.com/mullenlucas/currency-converter"
            >
               
              GitHub repository
               
            </a>
            for more information and acknowledgements
          </p>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer>
      <div
        class="footer-header"
      >
        <p>
          Lucas Mullen
        </p>
        <p>
          All rights reserved
        </p>
      </div>
      <div
        class="footer-footer"
      >
        <p>
          Go to
          <a
            href="https://github.com/mullenlucas/currency-converter"
          >
             
            GitHub repository
             
          </a>
          for more information and acknowledgements
        </p>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
